spec/spec_helper.rb

```diff
@@ -0,0 +1,16 @@
+ENV['RAILS_ENV'] ||= 'test'
+require_relative 'dummy/config/environment.rb'
+require 'pry'
+
+Rails.application.routes.default_url_options[:host] = 'localhost:3000'
+
+require 'tmpdir'
+ActiveStorage::Blob.service =
```

Comment:
Why do you need to do this? Can't you set it up in the dummy app the same way you'd do it in a real app?

Reply:
This is actually what Rails does in their tests. This is their [test_helper](https://github.com/rails/rails/blob/master/activestorage/test/test_helper.rb)